.docker/nginx.conf

```diff
@@ -0,0 +1,51 @@
+# Upstream to abstract backend connection(s) for php
```

Comment:
```suggestion
# Upstream to abstract backend connection(s) for PHP.
```

Reply:
Fixed in https://github.com/WordPress/wordcamp.org/pull/113/commits/1e3bc2e5e0ebcc4643db21c0d98a69b94a51b6c8